go.sum

```diff
@@ -0,0 +1,16 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
+github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/ibm-messaging/mq-golang v3.0.0+incompatible h1:Yc3c8emAyveT54uNDRMkgvS+EBAHeLNWHkc3hk5x+IY=
+github.com/ibm-messaging/mq-golang v3.0.0+incompatible/go.mod h1:qjsZDb7m1oKnbPeDma2JVJTKgyCA91I4bcJ1qHY+gcA=
+github.com/ibm-messaging/mq-golang-jms20 v0.0.0-20190929175954-aaf8fcfd717b h1:IRw4ZiqisQZxNx1gFBFl0dFvPMTeo0mhyg1rZ005fhc=
+github.com/ibm-messaging/mq-golang-jms20 v0.0.0-20190929175954-aaf8fcfd717b/go.mod h1:oUD94mdxOnBDiyLzyDofRX99qnWIjLFsZpGYA4gtWHI=
```

Comment:
Presumably these will also disappear once we are defining the go.mod for the correct module?

Reply:
It's now in sync with go.mod